dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/BaseTraceService.java

```diff
@@ -392,7 +396,7 @@ public synchronized void update(LogProviderConfig config) {
         commonConsoleLogHandlerUpdates();
 
         /*
-         * If messageFormat has been configured to 'basic' OR if consoleFormat is neither basic nor json (default to basic)
+         * If messageFormat has been configured to 'basic' OR if messageFormat is neither basic nor json (default to basic)
```

Comment:
OR if messageFormat -> OR if consoleFormat

Reply:
This should have been messageFormat in the first commit instead of consoleFormat